SECURITY.md

```diff
@@ -14,6 +14,11 @@ PaddlePaddle attaches great importance to security and privacy of model. This in
 
 These tools include adversarial example evaluation test, pseudo-natural environment robustness evaluation test, model reversing evaluation test, member inference evaluation test, sample denoising, adversarial training, privacy enhancement optimizer, etc.
 
+### Running untrusted models
+
+Always load and execute untrusted models inside a sandbox and be sure to know the security impacts.
+There are several ways in which a model could become untrusted. PaddlePaddle has enough features to impact on the system. (e.g. `pickle.load` malformed models to achieve arbitrary code execution). So we recommend when using the untrusted models, you need to carefully audit it and run PaddlePaddle inside a sandbox.
+
```

Comment:
suggested wording:
e.g. `paddle.load` uses [pickle](https://docs.python.org/3/library/pickle.html) implicitly, which may cause malformed models to achieve arbitrary code execution